Formula/minishift.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
+
+class Minishift < Formula
+  desc "Runs a single-node OpenShift cluster inside a VM"
+  homepage "https://minishift.io/"
+  url "https://github.com/minishift/minishift/archive/v1.1.0.tar.gz"
+  version "1.1.0"
+  sha256 "67cca4ec4767f2609eadb05f2fe3322592a070319caad2e3c477acbf68e0bff0"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+        :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    (buildpath/"src/github.com/minishift/minishift").install buildpath.children
+    ENV.prepend_create_path "PATH", buildpath/"bin"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/jteeuwen/go-bindata/go-bindata" do
+      system "go", "install"
+    end
+
+    cd "src/github.com/minishift/minishift" do
+      system "make"
+      prefix.install_metafiles
+    end
+
+    bin.install "bin/minishift"
+  end
+
+  test do
+    assert_equal "minishift v#{version}\n", shell_output("#{bin}/minishift version")
```

Comment:
Yes, they are the same software. There is even a PR removing it once this PR is merged: https://github.com/caskroom/homebrew-cask/pull/35760